pom.xml

```diff
@@ -132,6 +132,12 @@
 			<scope>compile</scope>
 		</dependency>
 		<dependency>
+			<groupId>org.apache.commons</groupId>
```

Comment:
If you do not mind, the way `commons-lang3` is used, is covered by Google Guava. Can we switch for Guava instead?